Makefile

```diff
@@ -1013,7 +1013,8 @@ endif
 # Packaging components
 #
 ##############################
-PACKAGE_TARGETS = package_installer package_all_compress
+PACKAGE_TARGETS = package_installer package_ground package_matlab
```

Comment:
I just tried this.. package_all_compress no longer works at toplevel so Jenkins is not happy.